src/runtime/components/nuxt-img.vue

```diff
@@ -34,7 +42,8 @@ export default {
       }
       return this.$img(this.src, this.nModifiers, this.nOptions)
     },
-    nSizes () {
+    /* eslint-disable no-undef */
+    nSizes (): ReturnType<typeof $Img['getSizes']> {
```

Comment:
We could also import the type directly. It will be excluded either way. Thoughts?

Reply:
I think while we are moving towards strict typing, makes sense to also care about naming internal interfaces

Reply:
892b3f9509b80727803a995dec282c62a6d2f511